src/use-swr.ts

```diff
@@ -157,7 +158,7 @@ function useSWR<Data = any, Error = any>(
   const shouldReadCache = config.suspense || !useHydration()
 
   // stale: get from cache
-  let [data, setData] = useState(shouldReadCache ? cacheGet(key) : undefined)
+  let [data, setData] = useState(config.initialData || (shouldReadCache ? cacheGet(key) : undefined))
```

Comment:
I like the idea, but I think `config.initialData` should be the fallback value. 

```suggestion
  let [data, setData] = useState((shouldReadCache ? cacheGet(key) : undefined) || config.initialData)
```